tests/test_buf.rs

```diff
@@ -36,6 +36,14 @@ fn test_get_u16() {
     assert_eq!(0x5421, buf.get_u16_le());
 }
 
+#[test]
+fn test_get_int() {
```

Comment:
What would you think of adding above this a couple of "simpler" cases, like what's reported in the issue? So that readers (such as myself 🫣 ) can understand more quickly what it's testing.

Reply:
Sounds like a good idea :smile: